src/views/v2/emails/purchase-receipt/body.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Event Tickets Emails: Purchase Receipt Body template
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/purchase-receipt/body.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version 5.5.9
+ *
+ * @since 5.5.9
```

Comment:
We ned to set the since and version to TBD to any new template we're introducing.